app/views/keywords/new.html.slim

```diff
@@ -0,0 +1,7 @@
+= form_with(model: keyword, local: true, url: keywords_path) do |form|
+  h5.mb-3 = t('keyword.upload_keywords_csv_file')
+  .form-group
+    = form.file_field :file, class: "form-control-file", required: true
```

Comment:
how about limiting the file type the user can choose to CSV only?
```ruby
= form.file_field :file, class: "form-control-file", required: true, accept: 'text/csv'
```

Reply:
Fixed in 03a73ee4f887294193fef2f584318a72e75d1e3e